README.md

```diff
@@ -307,6 +307,7 @@ Total deals: 110
 
 |  | Name | Description | Discount Code & Terms |
 | -- | ---| ------ | ------ |
+| 🚀 | [Rocketlaunch](https://www.rocket-launch.dev/) | The Beginner's Ultimate Boilerplate to Accelerate Your SaaS 🚀 | 20% OFF pro license with code **BF2024** |
```

Comment:
Please put your deal at the end of the table

Reply:
Done :)